SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/RegisterAutoDestroyActorsTest/RegisterAutoDestroyActorsTest.cpp

```diff
@@ -28,10 +28,8 @@ void ARegisterAutoDestroyActorsTestPart1::PrepareTest()
 		FRotator SpawnPositionRotator = FRotator(0.0f, 360.0f / NumVirtualWorkers, 0.0f);
 		for (int32 i = 0; i != NumVirtualWorkers; ++i)
 		{
-			ACharacter* Character = World->SpawnActor<ACharacter>(SpawnPosition, FRotator::ZeroRotator);
-			AssertTrue(IsValid(Character),
-					   FString::Printf(TEXT("Spawned ACharacter %s in worker %s"), *GetNameSafe(Character),
```

Comment:
maybe we should keep the debug info as a log or something

Reply:
added (with its own log category)